docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -243,10 +244,179 @@ You can get a listing of those mailbox workspaces as follows.
+
+```console
+$ KUBECONFIG=~/.kube/config kubectl version --context kind-guilder
+WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
+Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.0", GitCommit:"a866cbe2e5bbaa01cfd5e969aa3e033f3282a8a2", GitTreeState:"clean", BuildDate:"2022-08-23T17:44:59Z", GoVersion:"go1.19", Compiler:"gc", Platform:"darwin/amd64"}
+Kustomize Version: v4.5.7
+Server Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.2", GitCommit:"5835544ca568b757a8ecae5c153f317e5736700e", GitTreeState:"clean", BuildDate:"2022-09-22T05:25:21Z", GoVersion:"go1.19.1", Compiler:"gc", Platform:"linux/amd64"}
+
+$ KUBECONFIG=~/.kube/config kubectl get --context kind-guilder ns
+NAME                 STATUS   AGE
+default              Active   89m
+kube-node-lease      Active   89m
+kube-public          Active   89m
+kube-system          Active   89m
+local-path-storage   Active   89m
+```
+
+When you are confident, deploy the syncer as follows.
+
+```console
+$ KUBECONFIG=~/.kube/config kubectl apply --context kind-guilder -f edge-syncer.yaml 
+namespace/kcp-edge-syncer-sync-target-g-2s28jt1z created
+serviceaccount/kcp-edge-syncer-sync-target-g-2s28jt1z created
+secret/kcp-edge-syncer-sync-target-g-2s28jt1z-token created
+clusterrole.rbac.authorization.k8s.io/kcp-edge-syncer-sync-target-g-2s28jt1z created
+clusterrolebinding.rbac.authorization.k8s.io/kcp-edge-syncer-sync-target-g-2s28jt1z created
+role.rbac.authorization.k8s.io/kcp-edge-dns-sync-target-g-2s28jt1z created
+rolebinding.rbac.authorization.k8s.io/kcp-edge-dns-sync-target-g-2s28jt1z created
+secret/kcp-edge-syncer-sync-target-g-2s28jt1z created
+deployment.apps/kcp-edge-syncer-sync-target-g-2s28jt1z created
+```
+
+You might check that the syncer is running, as follows.
+
+```console
+$ KUBECONFIG=~/.kube/config kubectl get --context kind-guilder deploy -A
+NAMESPACE                                NAME                                     READY   UP-TO-DATE   AVAILABLE   AGE
+kcp-edge-syncer-sync-target-g-2s28jt1z   kcp-edge-syncer-sync-target-g-2s28jt1z   1/1     1            1           39s
+kube-system                              coredns                                  2/2     2            2           91m
+local-path-storage                       local-path-provisioner                   1/1     1            1           91m
+```
+
+### Connect florin edge cluster with its mailbox workspace
+
+Do the analogous stuff for the florin cluster.
+
+```console
+$ kubectl ws root:espw
+Current workspace is "root:espw".
+
+$ kubectl ws 4yqm57kx0m6mn76c-mb-bfa7098f-3345-47e6-be45-3e5c5802a234            
+Current workspace is "root:espw:4yqm57kx0m6mn76c-mb-bfa7098f-3345-47e6-be45-3e5c5802a234" (type root:universal).
+
+$ ../kcp/bin/kubectl-kcp workload edge-sync sync-target-f --syncer-image quay.io/kcpedge/syncer:dev-2023-04-18 -o edge-syncer.yaml
```

Comment:
I guess this is just a minor mistake.
It's not `../kcp/bin/kubectl-kcp` but `$KCPDIR/bin/kubectl-kcp`

Reply:
Right!  Thanks.  Fixed in the force-push to 6de4d7cd